studio-docs/source/contracts.mdx

```diff
@@ -160,8 +162,8 @@ The dialog detects all tag names that are used in your source variant's schema,
 
 Your contract will filter types and fields from its source variant according to the following rules:
 
-* **If the Included Tags list is empty**, the contract schema _includes_ each type and field _unless_ it's tagged with an _excluded_ tag.
-* **If the Included Tags list is non-empty**, the contract schema _excludes_ each type and field _unless_ it's tagged with an _included_ tag.
+* **If the Included Tags list is empty**, the contract schema _includes_ each type and object/interface field _unless_ it's tagged with an _excluded_ tag.
+* **If the Included Tags list is non-empty**, the contract schema _excludes_ each union type and object/interface field _unless_ it's tagged with an _included_ tag.
```

Comment:
Assuming I'm interpreting this right, I recommend moving the "Object and interface types..." line above to here as a sub-bullet with slight modifications:

"Each object and interface type is _included_ as long as _at least one_ of its fields is included (unless the type is explicitly excluded)"